common/src/main/java/earth/terrarium/olympus/client/components/Widgets.java

```diff
@@ -98,6 +101,30 @@ public static MapWidget map(State<MapRenderer> state, Consumer<MapWidget> factor
     }
 
     public static MapWidget map(State<MapRenderer> state) {
-        return map(state, map -> {});
+        return map(state, map -> {
+        });
+    }
+
+    public static <T> CompoundWidget radio(RadioState<T> state, Consumer<RadioBuilder<T>> builder, Consumer<CompoundWidget> factory) {
+        RadioBuilder<T> radioBuilder = new RadioBuilder<>(state);
+        builder.accept(radioBuilder);
+        factory.accept(radioBuilder.build());
+        return radioBuilder.build();
+    }
+
+    public static CompoundWidget tristateRadio(RadioState<TriState> state, Consumer<RadioBuilder<TriState>> builder, Consumer<CompoundWidget> factory) {
```

Comment:
im thinking maybe we keep this one only and then drop the old tristate and just if its needed people can recreate it due to how simple it is. Also create an overloaded method where consumers arnt needed